panic in argName

The issue came from the line:
```
		if a.Obj.Kind == ast.Var || a.Obj.Kind == ast.Con {
```

I did an output of "%#v %T" right before that and got:
```
&ast.Ident{NamePos:4893, Name:"tmpTest", Obj:(*ast.Object)(nil)} *ast.Ident
```

The object in question is a global variable or constant being printed from a test file.
Runnable example:

```
package main

import 	"github.com/y0ssar1an/q"
var panicInTests string
func main(){
    q.Q(panicInTests) // This is OK
}
```
```
package main

import (
	"testing"

	"github.com/y0ssar1an/q"
)

func TestPanicQ(t *testing.T) {
	q.Q(panicInTests) // This will panic
}
```

Full panic here:
```
go test
&ast.Ident{NamePos:102, Name:"panicInTests", Obj:(*ast.Object)(nil)} *ast.Ident
--- FAIL: TestPanicQ (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x113186f]

goroutine 6 [running]:
testing.tRunner.func1(0xc4200709c0)
	/usr/local/go/src/testing/testing.go:622 +0x29d
panic(0x115a360, 0x1239920)
	/usr/local/go/src/runtime/panic.go:489 +0x2cf
github.com/y0ssar1an/q.argName(0x122ada0, 0xc42000b020, 0x1224c26, 0x44)
	/Users/johnschnake/go/src/github.com/y0ssar1an/q/helpers.go:30 +0x10f
github.com/y0ssar1an/q.argNames.func1(0x1229da0, 0xc420013380, 0x11102c8)
	/Users/johnschnake/go/src/github.com/y0ssar1an/q/helpers.go:79 +0x16e
go/ast.inspector.Visit(0xc42000b0e0, 0x1229da0, 0xc420013380, 0x110220a, 0x0)
	/usr/local/go/src/go/ast/walk.go:373 +0x3a
go/ast.Walk(0x12297a0, 0xc42000b0e0, 0x1229da0, 0xc420013380)
	/usr/local/go/src/go/ast/walk.go:52 +0x66
go/ast.Walk(0x12297a0, 0xc42000b0e0, 0x12f6168, 0xc42000f120)
	/usr/local/go/src/go/ast/walk.go:196 +0x1eb7
go/ast.walkStmtList(0x12297a0, 0xc42000b0e0, 0xc42000f140, 0x1, 0x1)
	/usr/local/go/src/go/ast/walk.go:32 +0x89
go/ast.Walk(0x12297a0, 0xc42000b0e0, 0x1229d60, 0xc420018b10)
	/usr/local/go/src/go/ast/walk.go:224 +0x1f16
go/ast.Walk(0x12297a0, 0xc42000b0e0, 0x12f60d8, 0xc420018b40)
	/usr/local/go/src/go/ast/walk.go:344 +0xf63
go/ast.walkDeclList(0x12297a0, 0xc42000b0e0, 0xc42000b080, 0x2, 0x2)
	/usr/local/go/src/go/ast/walk.go:38 +0x89
go/ast.Walk(0x12297a0, 0xc42000b0e0, 0x1229ee0, 0xc42008ea00)
	/usr/local/go/src/go/ast/walk.go:353 +0x2bb1
go/ast.Inspect(0x1229ee0, 0xc42008ea00, 0xc42000b0e0)
	/usr/local/go/src/go/ast/walk.go:385 +0x4b
github.com/y0ssar1an/q.argNames(0x1224c26, 0x44, 0xa, 0x3, 0x3, 0x4c, 0x0, 0x0)
	/Users/johnschnake/go/src/github.com/y0ssar1an/q/helpers.go:82 +0x29d
github.com/y0ssar1an/q.Q(0xc42002c788, 0x1, 0x1)
	/Users/johnschnake/go/src/github.com/y0ssar1an/q/q.go:175 +0x17c
github.com/johnSchnake/qPanic.TestPanicQ(0xc4200709c0)
	/Users/johnschnake/go/src/github.com/johnSchnake/qPanic/main_test.go:10 +0x7f
testing.tRunner(0xc4200709c0, 0x118f018)
	/usr/local/go/src/testing/testing.go:657 +0x96
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:697 +0x2ca
exit status 2
FAIL	github.com/johnSchnake/qPanic	0.011s
```

--- a/helpers.go
+++ b/helpers.go
@@ -26,7 +26,10 @@ func argName(arg ast.Expr) string {
 	name := ""
 	switch a := arg.(type) {
 	case *ast.Ident:
-		if a.Obj.Kind == ast.Var || a.Obj.Kind == ast.Con {
+		switch {
+		case a.Obj == nil:
+			name = a.Name
+		case a.Obj.Kind == ast.Var, a.Obj.Kind == ast.Con:
 			name = a.Obj.Name
 		}
 	case *ast.BinaryExpr,

--- a/helpers_test.go
+++ b/helpers_test.go
@@ -336,6 +336,15 @@ func TestExtractingArgsFromSourceText(t *testing.T) {
 			},
 			want: "-1",
 		},
+		{
+			id: 16,
+			arg: &ast.Ident{
+				NamePos: 65,
+				Name:    "string",
+				Obj:     nil,
+			},
+			want: "string",
+		},
 	}
 
 	// We can test both exprToString() and argName() with the test cases above.

